book/src/key-management.md

```diff
@@ -76,36 +102,17 @@ There are three important directories in Lighthouse validator key management:
 	- Defaults to `~/.lighthouse/{network}/validators`
 - `secrets/`: since the validator signing keys are "hot", the validator process
 	needs access to the passwords to decrypt the keystores in the validators
-	dir. These passwords are stored here.
-	- Defaults to `~/.lighthouse/{network}/secrets` where `network` is the name of the network passed in the `--network` parameter (default is `mainnet`).
+	directory. These passwords are stored here.
+	- Defaults to `~/.lighthouse/{network}/secrets` 
+	
+where `{network}` is the name of the network passed in the `--network` parameter.
 
 When the validator client boots, it searches the `validators/` for directories
 containing voting keystores. When it discovers a keystore, it searches the
-`secrets/` dir for a file with the same name as the 0x-prefixed hex
-representation of the keystore public key. If it finds this file, it attempts
+`secrets/` directory for a file with the same name as the 0x-prefixed validator public key. If it finds this file, it attempts
 to decrypt the keystore using the contents of this file as the password. If it
 fails, it logs an error and moves onto the next keystore.
 
 The `validators/` and `secrets/` directories are kept separate to allow for
 ease-of-backup; you can safely backup `validators/` without worrying about
 leaking private key data.
-
-### Withdrawal Keypairs
```

Comment:
I think we should probably put a note here (or at the very top of this page) that the Lighthouse wallet will always generate keys with 0x00 withdrawal credentials, and that users will have to update them after their validator becomes active.

I think we should also _strongly recommend_ that people use the deposit-cli.

Reply:
Updated as per recommendation